app/models/cloud_object_store_container/operations.rb

```diff
@@ -0,0 +1,7 @@
+module CloudObjectStoreContainer::Operations
+  extend ActiveSupport::Concern
+
+  def raw_delete
```

Comment:
@durandom if I decided to have this function wrapped in a higher function like
```ruby
def delete
   raw_delete
end
```
what name would you take? Perhaps
```ruby
def instance_delete
   raw_delete
end
```

or maybe
```ruby
def operation_delete
   raw_delete
end
```
?
I don't want to override ActiveRecord methods but can't find a pattern to do this on other models.